content/en/docs/kubernetes/helm/collector.md

```diff
@@ -0,0 +1,385 @@
+---
+title: OpenTelemetry Collector Chart
+linkTitle: Collector Chart
+spelling:
+  cSpell:ignore statefulset kuberenetes filelogreceiver loggingexporter
+  cSpell:ignore filelogreceiver otlphttp sattributesprocessor kubelet
+  cSpell:ignore kubeletstatsreceiver sclusterreceiver sobjectsreceiver
+  cSpell:ignore hostmetricsreceiver
+---
+
+## Introduction
+
+The [OpenTelemetry Collector](/docs/collector) is an important tool for
+monitoring a Kubernetes cluster and all the services that in within. To
+facilitate installation and management of a Collector deployment in a Kubernetes
```

Comment:
```suggestion
facilitate installation and management of a collector deployment in a Kubernetes
```

Write "collector" in lower case as long as you are not writing "OpenTelemetry Collector"